### ISSUE TITLE
Libraries to appear in Build path is not in tab order

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/google-cloud-eclipse#1128]
In new project wizards you can't tab into the Libraries to appear in build path checkboxes.

**Comment by chanseokoh** [comment]
Not reproducible on Linux. We saw that this doesn't work on OSX, so maybe something should be done for OSX?
